lib/nimble_template/addons/mimic.ex

```diff
@@ -37,4 +38,45 @@ defmodule NimbleTemplate.Addons.Mimic do
 
     project
   end
+
+  defp edit_case(%Project{mix_project?: false} = project) do
+    Generator.inject_content(
+      "test/support/channel_case.ex",
+      """
+          quote do
+      """,
+      """
+            use Mimic
+
+      """
+    )
+
+    Generator.inject_content(
+      "test/support/conn_case.ex",
+      """
+          quote do
+      """,
+      """
+            use Mimic
+
+      """
+    )
+
+    Generator.inject_content(
+      "test/support/data_case.ex",
+      """
+          quote do
+      """,
+      """
+            use Mimic
+
+      """
+    )
+
+    project
+  end
+
+  defp edit_case(project) do
+    project
+  end
```

Comment:
Shall we put this in a single-line?

Reply:
Hi @bterone we don't use much single-line in the current template convention 😂 so I will keep it as its :D